gluon-mesh-vpn-wireguard/Makefile

```diff
@@ -11,7 +11,7 @@ define Package/gluon-mesh-vpn-wireguard
   SECTION:=gluon
   CATEGORY:=Gluon
   TITLE:=Support for connecting meshes via wireguard
-  DEPENDS:=+gluon-mesh-vpn-core +gluon-config-mode-core +micrond +kmod-gre +kmod-wireguard +wireguard-tools +kmod-udptunnel4 +curl +ca-bundle +ip-full
+  DEPENDS:=+gluon-mesh-vpn-core +gluon-config-mode-core +micrond +kmod-wireguard +wireguard-tools +kmod-udptunnel4 +curl +ca-bundle +ip-full
```

Comment:
I believe `curl` and `ca-bundle` were required for the calls to the funky http backend. Are they still required?
Also `kmod-udptunnel4` seems superfluous, since it is a dependency of kmod-wireguard.

Reply:
Somehow we need to either dynamically add the keys to our backend or offer a regstration page so not sure yet if it‘s needed.